mks/operations.py

```diff
@@ -6,14 +6,19 @@
 from tma_saml import SamlVerificationException, UserType
 from urllib3.exceptions import ConnectTimeoutError
 
-from mks.model.stuf_utils import decrypt
+from mks.model.stuf_utils import decrypt, is_nil
 from mks.prometheus_definitions import mks_connection_state
-from mks.service import mks_client_02_04, adr_mks_client_02_04, mks_client_bsn_hr
+from mks.service import adr_mks_client_02_04, mks_client_02_04
 from mks.service.config import get_raw_key
-from mks.service.exceptions import NoResultException, InvalidBSNException, ExtractionError
-from mks.service.exceptions import ServiceException, onbekende_fout
-from mks.service.mks_client_bsn_hr import _get_from_mks
-from mks.service.saml import get_bsn_from_request, get_kvk_number_from_request, get_type
+from mks.service.exceptions import (ExtractionError, InvalidBSNException,
+                                    NoResultException, ServiceException,
+                                    onbekende_fout)
+from mks.service.mks_client_hr import (_get_response_by_bsn,
+                                       _get_response_by_kvk_number,
+                                       get_from_bsn, get_from_kvk,
+                                       get_nnp_from_kvk)
+from mks.service.saml import (get_bsn_from_request,
+                              get_kvk_number_from_request, get_type)
```

Comment:
die importer van jou is raar :p Ik heb nog nooit zo een import statement gezien.

Reply:
Zo doet mijn Python sort imports command het. Ik vind import statements met een `\` overigens raar.